vertical-pod-autoscaler/pkg/admission-controller/Dockerfile

```diff
@@ -32,4 +32,3 @@ ARG TARGETARCH
 COPY --from=builder /gopath/src/k8s.io/autoscaler/vertical-pod-autoscaler/pkg/admission-controller/admission-controller-$TARGETARCH /admission-controller
 
 ENTRYPOINT ["/admission-controller"]
-CMD ["--v=4", "--stderrthreshold=info"]
```

Comment:
It seems that adding flag variable re-use in a common place and cleaning up the static arguments in the various Dockerfiles are separate changes. Should there be a separate PR for each?

Reply:
I can separate this to two PRs if you prefer I don't mind

Reply:
I think that makes the most sense, thanks!

Reply:
Given that the point of this PR is to move defaults out of the Dockerfile into code, does it make sense to re-add the Dockerfile change, making the PR more atomic?

Reply:
@jackfrancis just want to see if you're fine with the Dockerfile change happening in this PR now. This PR is basically moving defaults out of the Dockerfile into the code.